flickr.py

```diff
@@ -25,18 +25,17 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr(tags):
     if not FLICKR_API_KEY:
         raise Exception('Flickr API Key is missing!')
 
     baseurl = "https://api.flickr.com/services/rest/"
     params_diction = {
         "method": "flickr.photos.search",
-        "format": "json",
         "api_key": FLICKR_API_KEY,
         "tags": tags,
         "per_page": 10,
-        "nojsoncallback": 1
+        "photo_id": photo_id,
```

Comment:
@amisbah Nice! Glad we are using `photo_id `here. 👍 🔥

Reply:
Thanks!